WebRoot/public/login.jsp

```diff
@@ -560,6 +562,7 @@ if (application.getInitParameter("offlineMode") != null) {
                                                 zimbraPasswordMinPunctuationChars = acct.getPasswordMinPunctuationChars();
                                                 zimbraPasswordMinNumericChars = acct.getPasswordMinNumericChars();
                                                 zimbraPasswordMinDigitsOrPuncs = acct.getPasswordMinDigitsOrPuncs();
+                                                zimbraPasswordAllowUsername = Boolean.toString(LC.allow_username_within_password.booleanValue());
```

Comment:
not sure why do we need direct LC access here, it should come from some SOAP api

Reply:
on this file, all attributes are obtained by calling Java methods directly. This particular attribute is different from others because it's stored in the LocalConfig class. This was validated with a backend peer.

Reply:
@BhavinBathani Can you please review this?

Reply:
LGTM